### PR TITLE
[10.0][T991] - Added website_sale_comment_category_wise module

### PR DESCRIPTION
**Pull request by YogeshMahera-SerpentCS**
-  TASK #[991](https://www.quartile.co/web?#id=991&action=771&model=project.task&view_type=form&menu_id=505)

### CODE REVIEW

**Comment by YogeshMahera-SerpentCS** [comment]
download

**Comment by YogeshMahera-SerpentCS** [comment]
Hi @TimLai125 the Travis is going to fail due to other modules

**Review by TimLai125 — CHANGES_REQUESTED** [review]
Please remove the <data> tag in the xml files.

**Comment by YogeshMahera-SerpentCS** [comment]
Hi @TimLai125 Changes amended.

File: website_sale_comment_category_wise/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# Copyright 2020 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+{
+    'name': 'Website Sale Comment Category wise',
+    'version': '10.0.1.0.0',
+    'author': 'Quartile Limited',
+    'website': 'https://www.quartile.co',
+    'category': 'Website',
+    'license': "LGPL-3",
+    'description': """    """,
```
Comment (TimLai125) [inline website_sale_comment_category_wise/__manifest__.py]: Remove `description`

File: website_sale_comment_category_wise/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+from . import controllers
```
Comment (TimLai125) [inline website_sale_comment_category_wise/__init__.py:3]: Add an empty line above `from . import controllers`

File: website_sale_comment_category_wise/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    'depends': [
+        'website_sale',
+    ],
+    'data': [
+        'views/templates.xml',
+        'views/product_public_category_view.xml',
```
Comment (TimLai125) [inline website_sale_comment_category_wise/__manifest__.py]: product_public_category_views.xml, please update the filename as well.

File: website_sale_comment_category_wise/controllers/__init__.py [hunk 4]
```diff
@@ -0,0 +1,2 @@
+# -*- coding: utf-8 -*-
+from . import main
```
Comment (TimLai125) [inline website_sale_comment_category_wise/controllers/__init__.py:3]: Add an empty line above from . import controllers

File: website_sale_comment_category_wise/controllers/main.py [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Comment (TimLai125) [inline website_sale_comment_category_wise/controllers/main.py]: LGPL

File: website_sale_comment_category_wise/models/product_public_category.py [hunk 6]
```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Comment (TimLai125) [inline website_sale_comment_category_wise/models/product_public_category.py]: LGPL

File: website_sale_comment_category_wise/views/product_public_category_view.xml [hunk 7]
```diff
@@ -0,0 +1,12 @@
+<odoo>
```
Comment (TimLai125) [inline website_sale_comment_category_wise/views/product_public_category_view.xml]: I think it is better to add `<?xml version="1.0" encoding="utf-8"?>` in v10.0

File: website_sale_comment_category_wise/views/templates.xml [hunk 8]
```diff
@@ -0,0 +1,20 @@
+<odoo>
```
Comment (TimLai125) [inline website_sale_comment_category_wise/views/templates.xml:2]: I think it is better to add `<?xml version="1.0" encoding="utf-8"?>` in v10.0

**Review by TimLai125 — CHANGES_REQUESTED** [review]
@YogeshMahera-SerpentCS Sorry for some comments may be a bit picky, but we want to keep the style nice and united.

**Comment by YogeshMahera-SerpentCS** [comment]
Hi @TimLai125 Changes amended.

File: website_sale_comment_category_wise/views/templates.xml [hunk 9]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?
```
Comment (TimLai125) [inline website_sale_comment_category_wise/views/templates.xml]: `<?xml version="1.0" encoding="utf-8"?>`

**Review by TimLai125 — CHANGES_REQUESTED** [review]

**Comment by YogeshMahera-SerpentCS** [comment]
Tested in local. working nicely 😊

**Comment by TimLai125** [comment]
@YogeshMahera-SerpentCS , Please run pre-commit on this PR.

**Comment by YogeshMahera-SerpentCS** [comment]
Hi, @TimLai125 Actually the Travis is going to fail due to other modules.

**Comment by YogeshMahera-SerpentCS** [comment]
Hi, @TimLai125 for test purpose I removed other modules and keep the only one module tried to run the pre-commit file, So it successfully passed. Let me know if something miss?
rsz_screenshot_from_2020-06-03_12-31-02

**Comment by TimLai125** [comment]
> Hi, @TimLai125 for test purpose I removed other modules and keep the only one module tried to run the pre-commit file, So it successfully passed. Let me know if something miss?
> rsz_screenshot_from_2020-06-03_12-31-02

@YogeshMahera-SerpentCS This sounds strange, I expect the pre-commit command should at least complain about the missing readme files?

**Comment by YogeshMahera-SerpentCS** [comment]
Hi, @TimLai125 yes Added Readme files. and I will also investigate the issue of why it's not getting a warning.

**Pull request merged**